tests/test_asciigraf.py

```diff
@@ -154,3 +155,34 @@ def test_line_lengths():
         ("n1", "n2", OrderedDict([("length", 10)])),
         ("n1", "n4", OrderedDict([("length", 3)])),
     ]
+
+
+def test_line_labels():
+    graph = graph_from_ascii("""
+        A---(nuts)----B----(string)---C
+                      |
+                      |
+                      |
+                      D---(string)----E
+
+    """)
+
+    assert ("A", "B") in graph.edges()
+    assert ("A", "(ABC)") not in graph.edges()
```

Comment:
what's the reason for checking that a link between `A` and `(ABC)` does not exist?